rotkehlchen/data_migrations/manager.py

```diff
@@ -19,6 +20,7 @@ class MigrationRecord(NamedTuple):
 
 MIGRATION_LIST = [
     MigrationRecord(version=1, function=data_migration_1),
+    MigrationRecord(version=2, function=data_migration_2),
```

Comment:
I believe the test for migrations will fail because It was expecting here to have only one migration. I changed it when I did mine for version 2. Instead of checking that the version was 1 has to check that is `len(MIGRATION_LIST)`

Reply:
Hmm you are right. Damn.